development/local-logging-pipeline.md

```diff
@@ -0,0 +1,96 @@
+# Running a logging pipeline locally with Docker
```

Comment:
This needs to be linked somewhere so that it can be found. I'd also remove this sentence "There is no need to setup your local environment for C development
thanks to Docker."

And remove "development" from the file path. 

These are instructions for testing locally using docker.

Reply:
Sure.

Reply:
Pardon the force push, thanks.